scripts/src/java/org/oppia/android/scripts/todo/ClosedIssueCheck.kt

```diff
@@ -0,0 +1,117 @@
+package org.oppia.android.scripts.todo
+
+import org.oppia.android.scripts.todo.data.Todo
+import java.io.File
+
+/**
+ * Script for ensuring that all TODOs of the closed issue are resolved.
+ *
+ * Usage:
+ *   bazel run //scripts:closed_issue_check -- <path_to_directory_root> <closed_issue_number>
+ *   <github_sha>
+ *
+ * Arguments:
+ * - path_to_directory_root: directory path to the root of the Oppia Android repository.
+ * - closed_issue_number: issue number of the closed issue.
+ * - github_sha: sha of the latest commit on the develop branch.
+ *
+ * Example:
+ *   bazel run //scripts:closed_issue_check -- $(pwd) 6 3898188
```

Comment:
Small nit, but maybe use a real hash of the develop branch (can be shortened).

Reply:
We are using a 40 char hash in the workflow, so I think it would be really better if we also use it here too.
Done based on this.